content/articles/intro-to-emberjs/index.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: engineering-education
+status: publish
+published: false
+url: /intro-to-emberjs
+title: Introduction to Ember JS
+description: Ember.js is a free JavaScript web framework that follows the Model–View–View Model (MVVM) architecture. By combining popular idioms and best practises into the framework, developers may design scalable single-page web apps.
+author: dev-verma
+date: 2022-03-08 T00:00:00-13:23
+topics: [languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/intro-to-emberjs
+    alt: Ember JS
+---
+
+
+###  Ember JS
+### Introduction
+Ember.js is a free JavaScript client-side framework for designing web apps that is open source. It is a component-service framework that focuses on the overall online application development experience, eliminating minor variations between apps while remaining a current and lightweight overlay on top of native JavaScript. What does a component-service framework mean? Components, like those provided by other front-end frameworks like React, Vue, and Angular, are separate bundles of action, style, and HTML.
```

Comment:
Add link for Emberjs

Reply:
`component-service framework`

Brief about this.

PS:  `What does a component-service framework mean? Components, like those provided by other front-end frameworks`

What are components? Add 1 - 2 lines of explanation. And, brief about the framework. Add any reference links too

Reply:
`eliminating minor variations between apps while remaining a current`

State those minor variations.
Probably, you can add this as a new sub topic.